### PR TITLE
Update readme and doc

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Trivial fixes for README.rst and sphinx documentation settings

### CODE REVIEW

File: rpm/spec.in [hunk 1]
```diff
@@ -83,5 +83,5 @@ mkdir -p $RPM_BUILD_ROOT/
 %{_includedir}/libdrizzle-5.1/visibility.h
 
 %changelog
-* Wed Oct 11 2012 Brian Aker <brian@tangent.org> - 0.1-1
-- Initial package
+* Wed April 1 2016 Ben Palper <ben.palmer@sociomantic.com> - 5.2.0
```
Comment (ben-palmer-sociomantic) [inline rpm/spec.in]: haha `Ben Palper`

**Comment by andreas-bok-sociomantic** [comment]
My bad, hehe.

**Comment by andreas-bok-sociomantic** [comment]
Fixed typo

**Comment by andreas-bok-sociomantic** [comment]
Updated

``` diff
diff --git a/rpm/spec.in b/rpm/spec.in
index fec3b46..31e2985 100644
--- a/rpm/spec.in
+++ b/rpm/spec.in
@@ -83,5 +83,5 @@ mkdir -p $RPM_BUILD_ROOT/
 %{_includedir}/libdrizzle-5.1/visibility.h

 %changelog
-* Wed April 1 2016 Ben Palper <ben.palmer@sociomantic.com> - 5.2.0
+* Wed April 1 2016 Ben Palmer <ben.palmer@sociomantic.com> - 5.2.0
 - Project Moving
```

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Comment by ben-palmer-sociomantic** [comment]
This pull request has been rebased via `git hub pull rebase`. Original pull request HEAD was f9c34c1afb68c8dd09f97f9b359cafbe7fae7b65, new (rebased) HEAD is a38a7bc5d6181e321eea6f2ee5f0333b829ffac6

**Comment by andreas-bok-sociomantic** [comment]
TX!!

**Pull request closed without merging**